flags/flags.go

```diff
@@ -379,6 +379,19 @@ be stored in the 'sshpop' header.`,
 be stored in the 'nebula' header.`,
 	}
 
+	// Confirmation is a cli.Flag used to add a confirmation claim in the token.
+	Confirmation = cli.StringFlag{
+		Name:  "cnf",
+		Usage: `The <fingerprint> of the CSR to restrict this token for.`,
+	}
+
+	// ConfirmationFile is a cli.Flag used to add a confirmation claim in the
+	// tokens. It will add a confirmation kid with the fingerprint of the CSR.
+	ConfirmationFile = cli.StringFlag{
+		Name:  "cnf-file",
+		Usage: `The CSR <file> to restrict this token for.`,
+	}
+
```

Comment:
Should these include something about the SSH public key, or want to keep it simple?

Reply:
They used to include it, but due to a [comment](https://github.com/smallstep/certificates/pull/1660#discussion_r1547435804) in the certificates PR that suggested removing the cnf for SSH, I've removed it from here.